common/source/docs/common-benewake-tf02-lidar.rst

```diff
@@ -33,7 +33,7 @@ If the SERIAL4/5 port on a Pixhawk is being used then the following parameters s
 
 -  :ref:`SERIAL4_PROTOCOL <SERIAL4_PROTOCOL>` = 9 (Lidar)
 -  :ref:`SERIAL4_BAUD <SERIAL4_BAUD>` = 115 (115200 baud)
--  :ref:`RNGFND1_TYPE <RNGFND1_TYPE>` = 19 (Benewake TF02)
+-  :ref:`RNGFND1_TYPE <RNGFND1_TYPE>` = 19 (Benewake TF02), 20 (Benewake TF03 ArduCopter 3.6.12), 27 (Benewake TF03 ArduCopter 4)
```

Comment:
- labels must match the actual metadata shown for param
- PR has been submitted to change labeling for other issues including TFminiPlus I2C....
options at this point:
1. Close this, and I will issue a PR with all the new changes and correct new lablels...
2. I can make the changes directly to this PR and push to your branch so that they appear here to be merged when the code label PR is merged in a day or two...

your choice...let me know, and thanks for finding this issue on the TF03 documentation!

Reply:
Please make the changes directly to this PR (point 2). Thanks for your work. Sorry for messing in this whole github. I am trying to learn and understand how this work.

Reply:
not at all, please do not hesitate to try to improve the docs....using the web editing interface is easy, as you have found out....almost nothing submitted goes in without some modifications...dont worry...